src/main/scala/sbtsonar/SonarLoggerAdapter.scala

```diff
@@ -0,0 +1,20 @@
+package sbtsonar
+
+import org.sonarsource.scanner.api.LogOutput
+import org.sonarsource.scanner.api.LogOutput.{Level => SonarLogLevel}
+import sbt.{Level, Logger}
+
+class SbtSonarLoggerAdapter(implicit logger: Logger) extends LogOutput {
+  override def log(formattedMessage: String, level: SonarLogLevel): Unit = {
```

Comment:
I don't think this is true... we're propagating all sonar.* properties to EmbeddedScanner so any overrides should cause it to log at the correct levels. Those levels are then mapped to SBT logger statements at the given levels which are shown/suppressed according to how SBT's logging is configured.

Reply:
That's what I thought initially, but it looks to me that `EmbeddedScanner` wraps your implementation of `LogOutput` in a `LoggerAdapter` ([here](https://github.com/SonarSource/sonar-scanner-api/blob/4ba28713442f1074d4283fd72cd25518eb6dbbc2/api/src/main/java/org/sonarsource/scanner/api/EmbeddedScanner.java#L58)), which just proxies all log calls to your implementation of the `log` function without dealing with any properties (see [here](https://github.com/SonarSource/sonar-scanner-api/blob/4ba28713442f1074d4283fd72cd25518eb6dbbc2/api/src/main/java/org/sonarsource/scanner/api/LoggerAdapter.java)), so we need to implement this logic ourselves. In fact that's exactly how it's done in sonar-scanner-cli, [where](https://github.com/SonarSource/sonar-scanner-cli/blob/050dd2d15cf83feb44857ffec5fe0b6073797bf7/src/main/java/org/sonarsource/scanner/cli/Main.java#L102) they specifically enable debug level based on the provided sonar properties.

Reply:
Ideally I think that propagating SBT's logger level to the Sonar properties would be the cleanest solution.
It would prevent Sonar from doing unnecessary work to log at levels which are subsequently discarded.
However, I have not found a way to programatically query an SBT logger to determine the current level.
I'm not terribly familiar with SBT internals so perhaps I've missed something.

Reply:
> Ideally I think that propagating SBT's logger level to the Sonar properties would be the cleanest solution.

Wouldn't it be the case of getting the current value from the `logLevel` sbt key?